src/System.Private.Uri/src/System/UriBuilder.cs

```diff
@@ -114,19 +81,12 @@ string extraValue
             {
                 Extra = extraValue;
             }
-            catch (Exception exception)
+            catch (ArgumentException)
```

Comment:
I don't understand this change.  How is this equivalent?  What if some other exception happened to emerge from set_Extra, such as an IndexOutOfRangeException or something like that?